src/js/hca-rjsf/helpers.jsx

```diff
@@ -0,0 +1,10 @@
+import { transformForSubmit } from '../common/schemaform/helpers';
+
+export function transform(formConfig, form) {
+  const formData = transformForSubmit(formConfig, form);
+  return JSON.stringify({
+    educationBenefitsClaim: {
```

Comment:
Taking a wild guess that it has something to do with what the backend will expect, though I'm not seeing it in https://github.com/department-of-veterans-affairs/vets-api/tree/master/docs/api . @annekainicUSDS, is that because we're not yet using the vets-api backend?